TROUBLESHOOTING.md

```diff
@@ -80,9 +80,13 @@ To read the error message returned by SendGrid's API:
   request.path = '/v3/mail/send'
   request.body = requestBody
   sg.API(request, function (error, response) {
-    console.log(response.statusCode)
-    console.log(response.body)
-    console.log(response.headers)
+    
```

Comment:
I would also like to keep the logging of the response in the case of no error for demonstration purposes. Could you please add that back in? Perhaps just add an `else`. Thanks!